File: Java1/Solutions/Doubles/src/doubles/Doubles.java

package doubles;

import java.util.Scanner;

public class Doubles {

   
    public static void main(String[] args) {
        Scanner keyboard = new Scanner(System.in);
        int dice1=0, dice2=1;
        while (dice1 != dice2) {
            System.out.println("Want to roll the dice?");
            String answer = keyboard.nextLine();
            if (!(answer.equals("yes") || answer.equals("y"))) {
                break;
            }
            dice1 = (int )(Math.random() * 6 + 1);
            dice2 = (int )(Math.random() * 6 + 1);
            System.out.println("You rolled a " + dice1 + " and a " + dice2 + "!");
            if (dice1==dice2) {
                System.out.println("Doubles! Yay!");
            }
        }

    }
    
}

File: Python/Solutions/Doubles.py
import random
dice1=0
dice2=1
while (dice1 != dice2):
    print "Want to roll the dice? Type yes or y."
    answer = str(raw_input())
    if ((answer!="yes") & (answer!="y")):
        break
    dice1 = random.randint(1,6)
    dice2 = random.randint(1,6)
    print "You rolled a " + str(dice1) + " and a " + str(dice2) + "!"
    if (dice1==dice2):
        print "Doubles! Yay!"